packages/dashboard/src/dashboard-actions/dashboardReducer.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { dashboardReducer } from './dashboardReducer';
+import { onMoveAction } from './actions';
+import { Position } from '../types';
+
+describe('dashboardReducer', () => {
```

Comment:
were already in the `dashboardReducer` scope since were in the `dashboardReducer` test suite, there's no need for a describe block that repeats that